### PR TITLE
Load database schema from sql

### PR DESCRIPTION
**Pull request by faun**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ bundler_args: "--without development --path=~/.bundle"
 before_script:
 - cp config/database.yml.travis config/database.yml
 - psql -c 'create database travis_ci_test;' -U postgres
-- bundle exec rake db:schema:load db:migrate RAILS_ENV=test
+- psql -U postgres -q -d travis_ci_test -f db/structure.sql
```
Comment (ggilder) [inline .travis.yml]: `bundle exec rake db:structure:load`

File: db/schema.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@
   enable_extension "plpgsql"
 
   create_table "authors", force: :cascade do |t|
-    t.string   "name"
+    t.string   "name",       limit: 255
```
Comment (ggilder) [inline db/schema.rb]: You shouldn't have this file at all if you're using `structure.sql`

Comment (faun) [inline db/schema.rb]: Good point

File: .travis.yml (hunk 1 above)
Comment (faun) [inline .travis.yml]: I was trying to see if I could shave some time off the build by not loading rake at all, but it's currently a difference of maybe 5 seconds.

**Comment by faun** [comment]
@ggilder Does this seem worth it? I prefer to have the schema format in Rubby and 5 seconds doesn't seem like a huge gain. However, if the Rails boot time increases, so does this step. WWGD?

**Comment by faun** [comment]
Should I close this PR or merge it?

**Comment by ggilder** [comment]
I think `structure.sql` is generally preferable; why would you prefer to keep it in rubbbbby?

**Comment by faun** [comment]
I have no good reasons other than readability and portability. Merging.

**Pull request merged**